azure-blob-storage/README.md

```diff
@@ -0,0 +1,33 @@
+# Prerequisites
+You  should have Node.js > 7.8 installed.
+
+## Installation
+```sh
+$ npm install
+```
+
+## Usage
+ ```sh
+$ node app.js [Options] -- <api_key> <api_secret> <funnel_id>
+```
+
+### Options
+- -c, --container=STRING  The name of the azure container
+- -s, --storageAccount=STRING   The name of the storage account
+- -t, --token=STRING  SAS token
+- -p, --filePrefix=STRING   Blobs prefix
+- -d, --daysBack      Search for blobs that modified in the last X days
```

Comment:
Maybe `days back to check`? what you wrote is the description of the check. Also, what about error/warning rate?